en/setup/dev_env_mac.md

```diff
@@ -50,6 +42,31 @@ brew install python3
 pip3 install --user pyserial empy toml numpy pandas jinja2 pyyaml pyros-genmsg
 ```
 
+## Gazebo Simulation
+
+To install SITL (software in the loop) simulation based using Gazebo you need to install XQuartz :
```

Comment:
Minor point, but I'd probably do this in one step, and not specifically mention the need for quartz

To install [SITL Simulation](LINK) with Gazebo:
```sh
brew cask install xquartz
brew install px4-sim-gazebo
```

If you prefer it this way, then review English where you say "based using "